src/Common/src/System/Security/Cryptography/DerEncoder.cs

```diff
@@ -841,7 +881,7 @@ private static bool IsPrintableStringCharacter(char c)
             return false;
         }
 
-        private static byte[] ConcatenateArrays(byte[][][] segments)
+        private static byte[] ConcatenateArrays(params byte[][][] segments)
```

Comment:
Why did this change to ```params```?  I might be missing it, but I don't see a call site that actually uses that.